ga4gh/protocol.py

```diff
@@ -11,22 +11,21 @@
 
 import google.protobuf.json_format as json_format
 import google.protobuf.message as message
+import google.protobuf.struct_pb2 as struct_pb2
```

Comment:
@calbach Maybe my eyes are playing tricks on me, but where is `struct_pb2` defined as I don't see it in the following directory - and it seems like it got removed in this commit:

https://github.com/google/protobuf/tree/master/python/google/protobuf

But you generate it separately here:

https://github.com/jeromekelleher/server/pull/4

Which got merged here:

https://github.com/jeromekelleher/server/blob/master/proto/google/protobuf/struct_pb2.py

But this one seem to have the following Travis failure:

https://travis-ci.org/jeromekelleher/server/builds/67432646

Thanks,
~p